### ISSUE TITLE
werkzeug version in requirements.txt

### ISSUE THREAD

**Issue by seem8** [getredash/redash#4643]
Since it's my first post here. Hi! :) 

### Issue Summary

I think, that werkzeug should have higher version in requirements.txt.

### Steps to Reproduce

1. On werkzeug 0.16.1
python3 manage.py 
Traceback (most recent call last):
  File "manage.py", line 6, in <module>
    from redash.cli import manager
  File "/home/lukasz/progi/redash/redash/redash/__init__.py", line 14, in <module>
    from .app import create_app  # noqa
  File "/home/lukasz/progi/redash/redash/redash/app.py", line 2, in <module>
    from werkzeug.middleware.proxy_fix import ProxyFix
ModuleNotFoundError: No module named 'werkzeug.middleware'

2. After uprading werkzeug to version 1.0.0:
python3 manage.py
Usage: manage.py [OPTIONS] COMMAND [ARGS]...
[...]
it works.

### Technical details:
I think it's more do to with werkzeug as a Flask dependency, but I'm not sure.

* Redash Version:
From today.
* Browser/OS:
None so far / Fedora 31
* How did you install Redash:
I pulled from git into virtualenv dir. 

Thanks. Redash looks like a really good application.

**Comment by arikfr** [comment]
Are you sure you got this error with werkzeug 0.16.1?

**Comment by seem8** [comment]
Yes it was 0.16.1. From pipdeptree:
[...]
Flask-Limiter==0.9.3
    - Werkzeug [required: >=0.15, installed: 0.16.1]
[...]
and I think I know what happened. manage.py throw an exception:
python3 manage.py 
Traceback (most recent call last):
  File "manage.py", line 6, in <module>
    from redash.cli import manager
  File "/home/lukasz/progi/redash3/redash/redash/__init__.py", line 56, in <module>
    import_query_runners(settings.QUERY_RUNNERS)
  File "/home/lukasz/progi/redash3/redash/redash/query_runner/__init__.py", line 263, in import_query_runners
    __import__(runner_import)
  File "/home/lukasz/progi/redash3/redash/redash/query_runner/cloudwatch.py", line 1, in <module>
    import boto3
ModuleNotFoundError: No module named 'boto3'

so... I installed flask-boto3, which pulled Flask 1.1.1, which than seeks for werkzeug.middleware module. After uprading werkzeug to 1.0.0 manage.py works, but pip complains about incompatibility;
ERROR: flask-boto3 0.3.2 has requirement werkzeug==0.14.1, but you'll have werkzeug 1.0.0 which is incompatible.

So yes. I figure it out and post requirements.txt, that will work for me.
Thanks.

**Comment by seem8** [comment]
I figured it out. I think my biggest problem was insisting of using Redash without containers (I know that is now officially supported now) and it works now. I installed it on a local VM and plan to rsync that to production server, to avoid installing a ton of dependencies for building Python modules and npm there. 
I used both requirements_all_ds.txt and requirements.txt file and installed everything.
Sorry for poking You with it.
